debian/README.md

```diff
@@ -8,23 +8,32 @@ The following packages are required to build AGS. The versions in
 parentheses are known to work, but other versions will also
 probably work.
 
--   Allegro 4 (>= 4.2.2)
--   libaldmb (0.9.3)
--   libdumb (0.9.3)
--   libfreetype (2.4.9)
+-   libsdl2 (2.0.12 or higher)
+-   libsdl_sound for sdl2 (revision 997e90562b35 or higher)
 -   libogg (1.2.2-1.3.0)
 -   libtheora (1.1.1-1.2.0)
 -   libvorbis (1.3.2)
--   libXext (1.3.3)
--   libXxf86vm (1.1.4)
 
 Fedora package installation
 ---------------------------
-    yum -y install git allegro-devel dumb-devel freetype-devel libogg-devel libtheora-devel libvorbis-devel libXext-devel libXxf86vm-devel
+    yum -y install git sdl2-devel libogg-devel libtheora-devel libvorbis-devel
 
 Debian/Ubuntu package installation
 ----------------------------------
-    sudo apt-get install git debhelper build-essential pkg-config libaldmb1-dev libfreetype6-dev libtheora-dev libvorbis-dev libogg-dev
+    sudo apt-get install git debhelper build-essential pkg-config libsdl2-dev libtheora-dev libvorbis-dev libogg-dev
+
+SDL_Sound library installation
+----------------------------------
+At the time of writing SDL_Sound 2.* did not have a proper release and almost no linux distro provides it.
+Until that is resolved, we recommend to download particular revision archive using following url:
+
+    https://hg.icculus.org/icculus/SDL_sound/archive/9262f9205898.tar.bz2
+
+Before building the library, perform following command on its source directory:
+
+    sed 's|FILES SDL_sound.h DESTINATION include|FILES src/SDL_sound.h DESTINATION include/SDL2|' -i CMakeLists.txt
```

Comment:
To be compatible with more sed implementations, this should probably be written as:
```
sed -i.bak 's|FILES SDL_sound.h DESTINATION include|FILES src/SDL_sound.h DESTINATION include/SDL2|' CMakeLists.txt
```
...assuming that the presence of the backup file doesn't cause a problem.

Reply:
This sed is unneeded in the latest code from mercurial.

Reply:
See : https://hg.icculus.org/icculus/SDL_sound/archive/997e90562b35.tar.gz

Reply:
Hmm interesting, this might be the relevant part of Makefiles.txt:

<pre>
install(TARGETS ${SDLSOUND_INSTALL_TARGETS}
        RUNTIME DESTINATION bin
        LIBRARY DESTINATION lib${LIB_SUFFIX}
        ARCHIVE DESTINATION lib${LIB_SUFFIX})
install(FILES src/SDL_sound.h DESTINATION include/SDL2)
</pre>

So it does install into SDL2 dir now.

Reply:
> For SDL_Sound I used the hint by rofl0r he posted here: #1096 (comment)

yep, this was fortunately fixed in exactly the manner i proposed, but there are other issues stil lingering: https://github.com/sezero/SDL_sound/commit/e2e479f4a9deaeae02c8ac3039860b3348adb48f#commitcomment-45911235 but it seems @sezero is working on it. :+1:

Reply:
@rofl0r , the version wrong when installing the default branch is because the version on the CMakeLists.txt is wrong (look for the word `SOVERSION` in it and you will see it's being set to 1 when it should be a 2, and the `VERSION` is being set to 1.9.0 instead of 2.0.0). Probably this will be updated once it's released only.